src/auro-menu.js

```diff
@@ -185,15 +185,53 @@ class AuroMenu extends LitElement {
     }
   }
 
+  /**
+   * Used for applying indentation to each level of nested menu.
+   * @private
+   */
+  handleNestedMenus(menu) {
+    const nestedMenus = menu.querySelectorAll(':scope auro-menu');
+
+    if (nestedMenus.length === 0) return;
+
+    nestedMenus.forEach((nestedMenu) => {
+      const options = nestedMenu.querySelectorAll(':scope > auro-menuoption');
+
+      options.forEach((option) => {
+        option.innerHTML = '<span class="nestingSpacer"></span>' + option.innerHTML;
```

Comment:
This brings back horrible memories of using spacer gifs...

Reply:
I get that. I thought about putting padding to the left of each nested menu but that breaks the hover/active styles which would no longer be the full width of the menu.

I don't have another solution to cause progressive indenting with each layer. Do you have a different solution we can use?